lib/bridge.js

```diff
@@ -732,8 +745,25 @@ Bridge.prototype._onEvent = function(event) {
             this.opts.registration.isUserMatch(event.user_id, true)) {
         return Promise.resolve();
     }
+    // m.room.tombstone is the event that signals a room upgrade.
+    if (event.type === "m.room.tombstone" && this._roomUpgradeHandler) {
+        this._roomUpgradeHandler.onTombstone(ev);
+        if (this.opts.RoomUpgradeHandler.consumeEvent) {
+            return Promise.resolve();
+        }
+    }
+    else if (event.type === "m.room.member" &&
```

Comment:
pick a bracket style and stick with it: either `}\nelse` or `} else`. Applies to catches too.

Reply:
`}\nelse` is what apparently what `ESLint` wants.

Reply:
I'm failing to see where abouts in my changes that it's inconsistent..

Reply:
line 190 of this file vs here

Reply:
Well spotted